src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -72,6 +74,31 @@ class AzureFeature(
                 validator = { it.boolean() },
                 canBeSimplifiedConfig = true
             ),
+            AuroraConfigFieldHandler(
+                AuroraAzureApimSubPart.ConfigPath.enabled,
+                defaultValue = false,
+                validator = { it.boolean(required = false) }
+            ),
+            AuroraConfigFieldHandler(
+                AuroraAzureApimSubPart.ConfigPath.path,
+                validator = { it.startsWithSlash(AuroraAzureApimSubPart.ConfigPath.path) }
+            ),
+            AuroraConfigFieldHandler(
+                AuroraAzureApimSubPart.ConfigPath.openapiUrl,
+                validator = { it.validUrl(required = false) }
+            ),
+            AuroraConfigFieldHandler(
+                AuroraAzureApimSubPart.ConfigPath.serviceUrl,
+                validator = { it.validUrl(required = false) }
+            ),
+            AuroraConfigFieldHandler(
+                AuroraAzureApimSubPart.ConfigPath.policies,
+                validator = { it.isListOrEmpty() }
```

Comment:
Litt forvirret her, men om deyt er tillatt med en tom liste, så er kanskje null også tillat? Men, regner med dette er et pattern i boober som du ikke akkurat har endret på nå...

Reply:
Det kunne ha stått `it.isListOrEmpty( required = false )`. Se også svar under.